apps/web/src/services/usePlanningActivitiesGap.tsx

```diff
@@ -0,0 +1,40 @@
+import { useState, useEffect } from 'react';
+import { API_ENDPOINT, REQUEST_METHOD } from '../common';
+import { useHttp } from './useHttp';
+import { usePlanningContext } from './usePlanningContext';
+
+export const usePlanningActivitiesGap = () => {
+  const {
+    state: { sessionId },
+    updateProceedToNext,
+  } = usePlanningContext();
+  const [initialValues, setInitialValues] = useState<any>({
+    data: [],
+  });
+  const { sendApiRequest, fetchData } = useHttp();
+
+  const handleSubmit = (values: any) => {
```

Comment:
We can remove this and clear out stuffs. But we'll merge this and work on these later